src/csproj/functions/csproj-utils.ps1

```diff
@@ -4,13 +4,35 @@ $script:ns = 'http://schemas.microsoft.com/developer/msbuild/2003'
 
 $script:types = @"
 using System.Xml;
+using System.IO;
+
 public class Csproj {
     public System.Xml.XmlDocument Xml {get;set;}
     public string Path {get;set;}
     public string FullName { get { return Path; } }
     public string Name {get;set;}
     public string Guid {get;set;}
+	public string Folder {get {return System.IO.Path.GetDirectoryName(Path);}}
+	public string TargetFw {get;set;}
     
+	public Csproj() {
+	}
+
+	public Csproj(string pa) {
```

Comment:
Import-csproj does basically the same.  Because it's not possible to easily reload C# types in Powershell, I wanted to limit the amount of code in C# classes to minimum.